database/sql/04-products.sql

```diff
@@ -190,7 +191,8 @@ INSERT INTO product (id, name, slug, temporal_duration, temporal_resolution, dss
     ('b206a00b-9ed6-42e1-a34d-c67d43828810','ndfd_conus_airtemp_01h', 'ndfd-conus-airtemp-01h',3600,3600,'NDFD-CONUS-TEMP','5fab39b9-90ba-482a-8156-d863ad7c45ad','8f51e5b5-08be-4ea7-9ebc-ad44b465dbc6', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus tincidunt nisl sit amet urna mattis, ac ornare sapien volutpat. Nullam laoreet finibus auctor. Donec nisi diam, porttitor et pharetra id, sollicitudin vestibulum dolor. Aliquam porttitor purus non massa ullamcorper, sit amet bibendum risus ornare. Sed ac metus tristique, iaculis arcu a, consequat augue. In id maximus purus. In euismod volutpat velit, a congue est tempor a.'),
     ('dde59007-25ec-4bb4-b5e6-8f0f1fbab853','ndfd_conus_airtemp_03h', 'ndfd-conus-airtemp-03h',10800,10800,'NDFD-CONUS-TEMP','5fab39b9-90ba-482a-8156-d863ad7c45ad','8f51e5b5-08be-4ea7-9ebc-ad44b465dbc6', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus tincidunt nisl sit amet urna mattis, ac ornare sapien volutpat. Nullam laoreet finibus auctor. Donec nisi diam, porttitor et pharetra id, sollicitudin vestibulum dolor. Aliquam porttitor purus non massa ullamcorper, sit amet bibendum risus ornare. Sed ac metus tristique, iaculis arcu a, consequat augue. In id maximus purus. In euismod volutpat velit, a congue est tempor a.'),
     ('f48006a5-ad25-4a9f-9b58-639d75763dd7','ndfd_conus_airtemp_06h', 'ndfd-conus-airtemp-06h',21600,21600,'NDFD-CONUS-TEMP','5fab39b9-90ba-482a-8156-d863ad7c45ad','8f51e5b5-08be-4ea7-9ebc-ad44b465dbc6', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus tincidunt nisl sit amet urna mattis, ac ornare sapien volutpat. Nullam laoreet finibus auctor. Donec nisi diam, porttitor et pharetra id, sollicitudin vestibulum dolor. Aliquam porttitor purus non massa ullamcorper, sit amet bibendum risus ornare. Sed ac metus tristique, iaculis arcu a, consequat augue. In id maximus purus. In euismod volutpat velit, a congue est tempor a.'),
-    ('b50f29f4-547b-4371-9365-60d44eef412e','wrf_columbia_precip','wrf-columbia-precip',3600,3600,'WRF-COLUMBIA','eb82d661-afe6-436a-b0df-2ab0b478a1af','e245d39f-3209-4e58-bfb7-4eae94b3f8dd', 'WRF Columbia precipitation data created for the entire Columbia River Basin'),
+    ('b50f29f4-547b-4371-9365-60d44eef412e','wrf_columbia_precip','wrf-columbia-precip',3600,3600,'WRF-COLUMBIA-PRECIP','eb82d661-afe6-436a-b0df-2ab0b478a1af','e245d39f-3209-4e58-bfb7-4eae94b3f8dd', 'WRF Columbia precipitation data created for the entire Columbia River Basin'),
```

Comment:
DSS F part doesn't need the parameter as that will be automatically added to the C Part of a DSS path.  I would keep it **WRF-COLUMBIA**

Reply:
Changed